design/code.md

```diff
@@ -81,7 +81,7 @@ to work, particularly on Mac. These live in [CommandMangler].
 ### Diagnostics
 
 During parsing, clang emits diagnostics through a `DiagnosticConsumer` callback.
-clangd's [StoreDiags] implementation converts them into [Diag] objects,
+clangd [StoreDiags] implementation converts them into [Diag] objects,
```

Comment:
I think this is an error, "clangd StoreDiags implementation" is a cluster of nouns I can't parse